### PR TITLE
Rate component

### PR DESCRIPTION
**Pull request by viniciushvc**
Context:
image

Component: 
image

### CODE REVIEW

File: src/components/rate/index.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    throw new Error('Rate value must be between 0 and 5');
+  }
+
+  const totalStars = [...Array(5)];
+
+  // pluralize the label
+  const pluralStar = value > 1 ? 'stars' : 'star';
+
+  // message to be read by screen readers
+  const a11yMessage = `${value} out of 5 ${pluralStar}`;
```
Comment (codeferreira) [inline src/components/rate/index.tsx:43]: You don't need to pluralize the sentence. The correct will be: "1 out of 5 stars"

**Review by codeferreira — CHANGES_REQUESTED** [review]
Small typo change

File: src/components/rate/index.tsx (hunk 1 above)
Comment (viniciushvc) [reply src/components/rate/index.tsx:43]: yep, you right. I will fix that

**Review by codeferreira — APPROVED** [review]
Everything looks great!

**Pull request merged**